src/components/ListRow/index.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { spacing, colors } from "../../theme";
+
+import Button from "../Button";
+import { Row, Column } from "../Grid";
+import { PrimaryText, SecondaryText, BoldText, Link } from "../Text";
+import { OverflowIcon } from "../Icons";
+import { mediumAndUp, largeAndUp } from "../../theme/mediaQueries";
+
+const variants = ["standard", "withLink"];
+
+const ListContainer = styled.div`
```

Comment:
missing:
- default background color
- border radius
- box shadow or borders for first and last of type